TEDAPI v3 draft

arnaudjardine opened the pull request:
Work collaboration with OP for new TEDAPI v3

arnaudjardine committed:
TEDAPI v3 draft

pcolast commented on modules/ROOT/attachments/specs/api.yaml:
```diff
@@ -0,0 +1,734 @@
+          example: example@email.eu
+        expectedPublicationDate:
+          type: string
+          description: The notice expected publication date
```
The notice expected publication date as assigned by the Publications Office based on the OJ S calendar

arnaudjardine replied:
Done

arnaudjardine committed:
Implement pcolast review comment

pcolast committed:
Update api.yaml

Changed some descriptions for existing operations

  /v3/notices/search was updated with definition from TED

pcolast reviewed (changes requested):
Dear Arnaud,
It would be helpful to keep the documentation for the existing PROD version (v2) and the next one for v3. For this reason we need to keep the existing files and not to delete them. Api.yaml was modified. Please review and include the modified file in the TEDAPI release. 
Thank you.
Stefan

bertrand-lorentz commented:
@pcolast : I guess this pull request will not be merged ? If that is the case, then please close it.

pcolast commented:
Closing this pull request as it will not be merged. 
api.yaml taken from the TEDAPI 2.0.0 release contains the latest changes.
